js/miniautures.js

```diff
@@ -0,0 +1,28 @@
+import  {urlForExport, likesForExport, commentsForExport} from './data.js';
+
+const pictures = document.querySelector ('.pictures');
+const pictureTemplate = document.querySelector ('#picture')
+  .content
+  .querySelector('a');
+
+const drawPicture = function () {
+  const newPicture = pictureTemplate.cloneNode(true);
+
+  const pictureSrcElement = newPicture.querySelector('src');
+  pictureSrcElement.textContent = urlForExport;
+  const pictureLikesElement = newPicture.querySelector('.picture__likes');
+  pictureLikesElement.textContent = likesForExport;
+  const pictureCommentsElement = newPicture.querySelector('.picture__comments');
+  pictureCommentsElement.textContent = commentsForExport;
```

Comment:
Текстовое содержимое элемента с количеством комментов устанавливаем как длина массива комментариев нашей картинки, которую получили в качестве аргумента функции (это и есть колчество комментов)
```suggestion
pictureCommentsElement.textContent = picture.comments.length;
```